.gitignore

```diff
@@ -57,3 +57,5 @@ superface/grid
 
 # Superface compiled capabilities
 grid/**/*.ast.json
+
+.idea
```

Comment:
This probably shouldn't be a part of this PR? :)

(Also, I assume you're using WebStorm for the TypeScript part?)